interface/themes/colors/openemr5/login.scss

```diff
@@ -8,6 +8,8 @@
   border-top: solid $dark 40px;
   margin-top: 5%;
   max-width: 500px;
+  border-radius: 10px;
+  box-shadow: 0 30px 60px 0 rgba($black,0.3);
 }
```

Comment:
hi @juggernautsei , travis is barking at this file not the one below :)

Reply:
@juggernautsei Yeah Travis is referring to this file not the one you're looking at